client/state/posts/selectors.js

```diff
@@ -414,7 +414,8 @@ export const isEditedPostDirty = createSelector(
 						return ! moment( value ).isSame( post.date );
 					}
 					case 'parent': {
-						return get( post, 'parent.ID', null ) !== value;
```

Comment:
Can't we just change the default value of `get` from `null` to `0`?

Reply:
Yes -- fixed.